### ISSUE TITLE
Cannot save view

### ISSUE THREAD

**Issue by steinm** [astubenbord/paperless-mobile#74]
I have created saved views in the past (it propably was version 1.4.0 or earlier). Now, with 1.5.1 it's not working anymore. There is not even a http access on the server and of course there is no error msg at all. I already restarted the app because sometimes it still happens that the app believes it is offline, but that doesn't help either.

**Comment by astubenbord** [comment]
I cannot reproduce this. Can you try to create a new empty view and see if that works? I just downloaded 1.5.1 from the play store and it's working as expected.

**Comment by steinm** [comment]
Creating a view without any filter parameter is actually working, but as soon as I add a filter (which is not a date), it cannot be saved anymore. I tried a term for title&content and also setting a tag. Suprisingly, filtering documents by date is working and that's what I did in the past. So maybe this bug already existed in early versions, but I just didn't encounter it, because I only created views for last week and last month.
